### ISSUE TITLE
References to GitHub or GitLab for Accessibility

### ISSUE THREAD

**Issue by mgifford** [HTTPArchive/almanac.httparchive.org#4192]
GitHub is making a big push to be more accessible. It would be interesting to know the accessibility of sites which are hosted on things like GitHub Pages. That should be possible to determine, even with custom domains. 

https://accessibility.github.com/

**Comment by tunetheweb** [comment]
Closing as wasn't included in the 2025 Accessibility chapter.
